docs/design/volume-snapshotting.md

```diff
@@ -1 +1,6 @@
-This file has moved to [https://github.com/kubernetes/community/blob/master/contributors/design-proposals/volume-snapshotting.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/volume-snapshotting.md)
+This file has moved to [/kubernetes/community/blob/master/contributors/design-proposals/volume-snapshotting.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/volume-snapshotting.md)
+
+
+<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
+[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/design/volume-snapshotting.md?pixel)]()
```

Comment:
@kerneltime after each rebase, I run `make update` to regenerate all the generated files, etc.  That script updated the docs at some points.  This is not something I manually changed. I will clean those changes and if rebase cleans it.